tracing-core/src/test/groovy/io/micronaut/tracing/instrument/util/MDCReactorSpec.groovy

```diff
@@ -1,5 +1,6 @@
 package io.micronaut.tracing.instrument.util
 
+import groovy.util.logging.Slf4j
```

Comment:
is it corret Slf4j? I think this is bad idea to add groovy to dependencies for this